assets/sass/index.sass

```diff
@@ -12,3 +13,54 @@ $pink-shadow: -2px 1px 1px #DB85B6, -2px 2px 1px #DD80B5, -4px 3px 1px #D77BAF
   font-size: 20px
   color: $white
   cursor: pointer
+
+.login-form
```

Comment:
could you please break styles by modules and include them in the main.sass? it's easier to maintain